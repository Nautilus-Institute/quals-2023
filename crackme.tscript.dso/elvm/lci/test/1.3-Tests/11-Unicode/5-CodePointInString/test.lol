HAI 1.3
	VISIBLE "Lorem :(0024) ipsum :(00A2) dolor :(20AC) sit :(024B62)"
KTHXBYE
