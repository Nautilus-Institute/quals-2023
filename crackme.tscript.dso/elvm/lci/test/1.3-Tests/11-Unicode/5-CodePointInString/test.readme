This test checks that when Unicode code points are inserted into strings, they
are translated correctly.
