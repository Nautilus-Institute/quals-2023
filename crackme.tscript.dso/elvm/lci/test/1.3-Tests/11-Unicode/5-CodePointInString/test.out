Lorem $ ipsum ¢ dolor € sit 𤭢
